packages/dds/tree/src/test/simple-tree/api/schemaCreationUtilities.spec.ts

```diff
@@ -102,6 +149,38 @@ describe("schemaCreationUtilities", () => {
 		}
 	});
 
+	it("adaptEnum - construction tests", () => {
+		const schemaFactory = new SchemaFactory("com.myApp");
+		enum Mode {
+			a = "A",
+			b = "B",
+			c = "C",
+		}
+		const ModeNodes = adaptEnum(new SchemaFactory(`${schemaFactory.scope}.Mode`), Mode);
```

Comment:
Is there a reason we need a separate schema factory here? I vaguely remember something about this, but it might be nice to add a comment explaining it.

Reply:
adaptEnum's docs recommend doing so to reduce the risk of collisions, so this is testing that.

Reply:
Nit: might be worth a quick comment noting that and referencing those docs.

Reply:
```suggestion
		// Uses a nested schema factory, as recommended by adaptEnum's docs to ensure that pattern works.
		const ModeNodes = adaptEnum(new SchemaFactory(`${schemaFactory.scope}.Mode`), Mode);
```